services/feeds.js

```diff
@@ -0,0 +1,160 @@
+var request = require('superagent');
+
+/**
+ * @constructor
+ */
+
+function Feeds(defaults) {
+
+	defaults = defaults || {};
+
+	/* set default format or lang for all requests */
+
+	/* we default to json since it's a js-sdk */
+	this.format = defaults.format || 'json';
+	this.lang = defaults.lang;
+}
+
+/**
+ * Creates an object with the option feed params
+ * for this request.
+ * @returns {Object}
+ */
+
+Feeds.prototype.params = function (args) {
+	var params;
+
+	args = args || {};
+
+	params = {
+		format: args.format || this.format
+	};
+
+	/* don't include unless explicitly set */
+	if (args.lang || this.lang) {
+		params.lang = args.lang || this.lang;
+	}
+	return params;
+};
```

Comment:
I'd rather not clog up the class namespace with methods or properties that don't describe the actual Feed API. Can we follow the pattern that request.js sets for the REST API?